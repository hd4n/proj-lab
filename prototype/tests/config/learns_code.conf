#kod_nev id agent_id aminocost nucleocost
@codes
co cd1 vi1 4 5

#mezo_nev id [tartalom]
@fields
la l1 cd1

#id akt_mezo [item_id1;item_id2...] [material_id1;material_id2...] [effect_id1;effect_id2...]
@virologists
v1 l1 null null null