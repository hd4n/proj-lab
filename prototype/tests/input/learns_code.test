load learns_code.conf
getstats v1
interact v1 readcode
getstats v1
getstats l1
