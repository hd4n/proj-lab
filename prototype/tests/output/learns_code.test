Load: sikeres

stats: v1
	fi_l1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null
interact: v1 l1

stats: v1
	fi_l1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_cd1
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null

stats: l1
	ci_v1
	ne_null
	co_cd1
