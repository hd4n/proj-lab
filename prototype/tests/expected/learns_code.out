Load: sikeres

stats v1:
fi_l1
eq_null
ma_null
ef_null
co_null

interact: v1 readcode

stats v1:
fi_l1
eq_null
ma_null
ef_null
co_cd1

stats l1:
vi_v1
eq_null
ma_null
co_cd1